src/utils/prompts.ts

```diff
@@ -13,6 +13,12 @@ export const createModel = (settings: ModelSettings) =>
     modelName:
       settings.customModelName === "" ? GPT_35_TURBO : settings.customModelName,
     maxTokens: 400,
+  },
+  {
+    basePath:
+      settings.customBaseUrl === ""
+        ? process.env.OPENAI_API_BASE_URL
```

Comment:
```suggestion
      settings.customBaseUrl === ""
        ? env.OPENAI_API_BASE_URL
```
Above we will need to import the client schema from `env/client.mjs`. Doing so allows us to preparse and validate the environment variables. More info can be found here: https://create.t3.gg/en/usage/env-variables